cli/command/service/update.go

```diff
@@ -627,14 +665,46 @@ portLoop:
 				continue portLoop
 			}
 		}
+
+		for _, portCSV := range removePorts {
+			portKVPairs, err := parsePortCSV(portCSV)
```

Comment:
It is not the entire list. Only `target-port` is mandatory. But the user can be more specific if they want to.